config/crd/bases/vpcresources.k8s.aws_cninodes.yaml

```diff
@@ -65,9 +64,3 @@ spec:
     served: true
     storage: true
     subresources: {}
-status:
```

Comment:
Not sure if removing the status is ok. Have you check if this is some cleanup added to v3?

Reply:
This is auto-generated, did not dig into it much. 
Let me check the details.

Reply:
Status was removed in CRD generation: https://github.com/kubernetes-sigs/controller-tools/pull/630
Issue: https://github.com/kubernetes-sigs/controller-tools/issues/456